lib/elixir/lib/inspect/algebra.ex

```diff
@@ -105,6 +105,50 @@ defmodule Inspect.Opts do
           syntax_colors: [{color_key, IO.ANSI.ansidata()}],
           width: non_neg_integer | :infinity
         }
+
+  @doc """
+  Builds an `Inspect.Opts` struct.
+  """
+  def new(opts) do
+    struct(%Inspect.Opts{inspect_fun: default_inspect_fun()}, opts)
+  end
+
+  @doc """
+  Returns the default inspect fun.
+  """
+  def default_inspect_fun do
```

Comment:
Nitpick: I would use "function" instead of "fun" for the function name, since we don't have "fun" in any other function name and we have `is_function`. There are `*_fun` as types though.

Reply:
Unfortunately we call it `inspect_fun` in Inspect.Opts. :(